### ISSUE TITLE
edit existing tags

### ISSUE THREAD

**Issue by sjjjjs** [notable/notable#594]
### Feature description

support edit existing tags

### Feature motivation

currently the app not support for editting existing tags, if we want change tag we can only delete and input new tags. but some times tags could be long words, it takes much time retype them.

**Comment by fabiospampinato** [comment]
Sounds like a good addition. Once autocomplete #20 is implement I guess there will probably be less of a need for this, but if the tag to edit is unique it may still be worth adding support for editing it directly.

**Comment by fmartelg** [comment]
I have been using Notable for a few weeks now, and have found myself in multiple situations where I had to delete tags, and recreate them as a way to "edit" them.  It is not just typing errors that autocomplete might solve.  Rather, wanting to reorganize files / hierarchies / rename things as the work progresses and takes shape.

**Comment by fabiospampinato** [comment]
@fmartelg yeah definitely, there's another issue tracking the addition of some convenience context menu actions for things that you could do today via the multi-note editor but which should be much easier to do: #480

**Comment by lilimelgar** [comment]
> I have been using Notable for a few weeks now, and have found myself in multiple situations where I had to delete tags, and recreate them as a way to "edit" them. It is not just typing errors that autocomplete might solve. Rather, wanting to reorganize files / hierarchies / rename things as the work progresses and takes shape.

Same here ;)

**Comment by zubko** [comment]
I'm also missing this feature a lot.

At least it's possible to use Sublime or smth to find/replace tags in the md files. 

Open data format rocks! 🚀🙂
